src/components/parking-component/ParkingComponent.tsx

```diff
@@ -1,7 +1,36 @@
-import React from "react";
+import { Grid } from '@material-ui/core';
+import Box from '@material-ui/core/Box'
+import React from 'react';
+import { ParkingMapService } from '../../services/ParkingMapService';
+import { ParkingPlace } from '../../services/ParkingPlace';
+import { ParkingPlaceComponent } from '../parking-place-component/ParkingPlaceComponent';
+import './ParkingComponent.css';
 
 export class ParkingComponent extends React.Component {
-    public render() {
-        return <div>Lorem ipsum cos tam</div>;
+
+    private parkingPlaces: ParkingPlace[][] = ParkingMapService.getParkingPlaces();
```

Comment:
For now I was mocking this functionality, when we will implement connection to backend we should do this other way, but in my opinion for now it will be ok.